src/binary-reversal/index.js

```diff
@@ -3,6 +3,29 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function binaryReversal(value) {
+  let binaryValue = Number(value).toString(2);
+  let lengthOfBinaryValue = binaryValue.length;
+
+  if (lengthOfBinaryValue === 8) {
+    return String(parseInt(
+      binaryValue
+        .split("")
+        .reverse()
+        .join(""),
+      2
+    ));
+  }
+
+  let lengthOfZeroToAdd = 8 - (lengthOfBinaryValue % 8);
+
+  let reverseBinaryValue = binaryValue.split("").reverse().join("");
+
+  let stringZeroToAdd = "0".repeat(lengthOfZeroToAdd);
```

Comment:
This is essentially `''.padStart(8, 0)`

Reply:
Thanks @bondz , i never knew there is a syntax like that